scripts/kconfig/kconfigfunctions.py

```diff
@@ -335,11 +335,52 @@ def dt_compat_enabled(kconf, _, compat):
     return "n"
 
 
+def dt_compat_get(kconf, _, prop, compat, inst, default=""):
```

Comment:
I'd rename function to less generic `dt_compat_get_instance`.
We could think `dt_compat_get` could be useful and later used to retrieve all instances.

Reply:
@ulfalizer gave me this explanation:
> Wonder if the function could just be called dt_compat_get too, since it also works for integers. Maybe there could be a dt_compat_get_array later if needed, that also takes an array index.

@erwango, I'm wonder if a function like **dt_compat_get_array** can address your concern?

Reply:
@nandojve, my point was on the nature of the result rather than its type.
Function seeks for one particular instance, seems logical to mention it, `dt_compat_get` by itself is misleading. 
Besides, since this function is intended to be used in Kconfig files, it is important that function name speaks by itself so reader does not need to reach function documentation.
Thinking about it, maybe `dt_compat_get_inst_prop` would be the best choice.

Reply:
I did rebase and change from 'dt_compat_get' to 'dt_compat_get_inst_prop'.